src/SFA.DAS.Forecasting.Domain/Estimations/AccountEstimationProjectionRepository.cs

```diff
@@ -34,12 +35,19 @@ public class AccountEstimationProjectionRepository : IAccountEstimationProjectio
         public async Task<IAccountEstimationProjection> Get(AccountEstimation accountEstimation)
         {
             var balance = await _currentBalanceRepository.Get(accountEstimation.EmployerAccountId);
-            var commitments = _commitmentModelListBuilder.Build(accountEstimation.EmployerAccountId, accountEstimation.VirtualApprenticeships);
+            var commitments = _commitmentModelListBuilder.Build(accountEstimation.EmployerAccountId, accountEstimation.Apprenticeships);
 
             var actualProjections = await _accountProjectionRepository.Get(accountEstimation.EmployerAccountId);
 
-            var employerCommitmentsModel = new EmployerCommitmentsModel();
-            employerCommitmentsModel.SendingEmployerTransferCommitments = commitments;
+            var employerCommitmentsModel = new EmployerCommitmentsModel
+            {
+                SendingEmployerTransferCommitments = commitments
+                    .Where(m => m.FundingSource == Models.Payments.FundingSource.Transfer || m.FundingSource == 0)
+                    .ToList(),
+                LevyFundedCommitments = commitments
+                    .Where(m => m.FundingSource == Models.Payments.FundingSource.Levy)
```

Comment:
Might be worth changing this to be anything other than funding source Transfer - as part of the work coming up is to bring in some of the other payment types for when the payment is co-investment

Reply:
Don't we want to be more explicit about what goes in to that list?
If I change it, would it still be **LevyFundedCommitments** then?

Reply:
Fair point, lets leave this for now as is 😄